### DISCUSSION TITLE
How can you get the length of a filtered dataframe?

### DISCUSSION THREAD

**Discussion by derrickcchow** [vaexio/vaex#1878] [Q&A] [answered]
Hello,
I am doing some manipulation of the dataframes and need to get the length of a filter data frame in order to do further manipulation to the data. 
The error: ValueError: Array is of length 10407, while the length of the DataFrame is 10407 due to the filtering, the (unfiltered) length is 34737.

Thanks, Derrick

**Comment by JovanVeljanoski** [comment, marked as answer]
Can you please provide a code example? Now we don't know what you've done. In general you should always provide a code example illustrating what fails. 

This should work:
```python
import vaex
df = vaex.example()
df2 = df[df.x>0]
print(len(df2))
```
